### PR TITLE
feat: build front end

### PR DESCRIPTION
**Pull request by Nekotoxin**

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Abingcbc plz review

File: flask/api/login.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def get(self):
         user = sdk.parse_jwt_token(token)
         session['casdoorUser'] = user
 
-        return redirect(request.root_url)
+        return jsonify({'status': 'ok'})
```
Comment (Abingcbc) [inline flask/api/login.py:37]: This will affect the template version of frontend

File: flask/api/account.py [hunk 2]
```diff
@@ -16,14 +16,17 @@
 from flask import current_app, jsonify, session
 from flask_restful import Resource
 
-from utils import authz_required
+from .utils import authz_required
 
 
 class Account(Resource):
 
-    @authz_required
```
Comment (Abingcbc) [inline flask/api/account.py]: Why `get-account` doesn't need authorization?

**Review by Abingcbc — COMMENTED** [review]
LGTM

**Pull request merged**